.github/workflows/wipac-cicd.yml

```diff
@@ -20,7 +20,15 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
-      - uses: WIPACrepo/wipac-dev-mypy-action@v1
+      # - uses: WIPACrepo/wipac-dev-mypy-action@v1.1
```

Comment:
Are we not running mypy anymore?

Reply:
```

Reply:
```
# there's a dependency clash with protobuf
# google wants protobuf>=4.21.3
# telemetry wants protobuf<4
```

Reply:
I wanted to make it work, but the mypy's "install everything" pattern (which I still think is best) fails. I'm hoping OpenTelemetry supports protobuf>=4.21.3 soon, hence the "6-month timer"

Reply:
Not sure if a timer is better than an open issue.  Could you at least make a better error message?  Maybe echo that comment to the shell.

Reply:
https://github.com/WIPACrepo/MQClient/issues/51

Reply:
@dsschult how about now? I added an echo referencing the GH issue